### DISCUSSION TITLE
Piggybank transactions change

### DISCUSSION THREAD

**Discussion by coldwar7** [firefly-iii/firefly-iii#4316] [Firefly III]
**Description**
I think it would be great if adding money to a piggy bank could be handled like a normal transaction. When you hit the plus or minus, you only get to put in the amount changed. I know you can link a transaction to a piggy bank but that is not how I am utilizing one of my piggy banks.

**Solution**
Make it so you can create a withdraw or deposit against a piggy bank without having to involve another account.

Ex. Savings account -> piggy bank

**What are alternatives?**
Just add the ability to include a note with a piggy bank addition or subtraction.

**Additional context**
I saw that all the necessary fields for this already exist in the database. Making them accessible in the UI for this use case would be great.

**My use case**
I have an "allowance" that I give myself every month. This is "do whatever I want" money for fast food, video games, etc. I use the budget feature to track this month to month. If I don't use the money, then I get to save it as still my pocket money. I have tried using the rolling budget feature but if I don't get everything into the database in time, the next month has the wrong starting funds for the budget. So now I take what I have left in the monthly budget when I know I've reconciled the month and I associate it with a piggy bank. This way it is controlled, but sometimes other things add to this money like getting paid for an odd job. I'd like to be able to see transaction notes along with these entries so I can track them.

**Comment by JC5** [comment]
No sorry, this is going to stay as it is for the time being. Piggy banks are meant to divide an asset account and tracking why money was added isn't a goal.
